Packs/Zscaler/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Zscaler
+- Fixed an issue where ***zscaler-category-add-url*** and ***zscaler-category-remove-url*** commands were failing on large categories.
```

Comment:
```suggestion
- Fixed an issue where the ***zscaler-category-add-url*** and ***zscaler-category-remove-url*** commands would fail on large categories.
```